{{ cookiecutter.format }}/briefcase.toml

```diff
@@ -5,6 +5,8 @@ target_version = "0.3.24"
 [paths]
 app_path = "src/app"
 app_packages_path = "src/app_packages"
+install_scripts_path = "src/scripts"
+license_path = "src/LICENSE.rtf"
```

Comment:
If the app uses `external_package_path`, as Constructor will, then there will be no `src` directory. And `scripts` isn't a good choice for the directory name, because that's one of the directories in the Conda base environment (the equivalent of `bin` on Unix).

One possible solution is to install the scripts to a directory with a name like `_installer` in the root of the installation tree, regardless of whether the app uses `external_package_path` or not. This would also be an appropriate place for the "bootstrap" script mentioned in https://github.com/beeware/briefcase/pull/2496#issuecomment-3369154367.

For `license_path`, I guess the file doesn't need to be in the installation tree, so maybe we can set it to just `LICENSE.rtf`?

Reply:
> install the scripts to a directory with a name like _installer in the root of the installation tree, regardless of whether the app uses external_package_path or not

As discussed, this means the installation tree would be produced by merging two locations. We could do this manually by copying them into a temporary directory, but maybe WiX can do this by adding additional `Files` or `File` elements.

Reply:
I've pushed an update that makes these changes. LICENSE.rtf is no longer in the source tree; `src` and `extras` are both collated for content to include in the MSI (with `src` being changed to the external package path for an external app).

Reply:
I've made both these changes. LICENSE is no longer included from inside the src directory; and the installer content is merged in from a second file root (`extras`). This way, the final installed content is the package path  (src, or the user-provided external path), plus the content in `extras` for the installer.